apps/storybook-react-native/.storybook/config.js

```diff
@@ -1,11 +1,8 @@
 import { addDecorator } from '@storybook/react-native';
 import { addons } from '@storybook/addons';
+import { useColorScheme } from 'react-native';
```

Comment:
Moved the usage of `useColorScheme` from inside the `ThemeProvider` to the Storybook config. This change delegates theme control to the consuming application, allowing it to determine the active theme based on the system color scheme. As a result, the internal logic of `ThemeProvider` is simplified, making it more flexible and easier to maintain.